test/Controllers/BookingsTest.php

```diff
@@ -0,0 +1,293 @@
+                    $rId, // resource_id
+                    NULL // user_id defaults to self
+                );
+            case 'update':
+                return new CreateBookingBody(
+                    date('Y-m-d', strtotime('tomorrow')), // date (within schedule start_date and end_date)
+                    $sId, // service_id
+                    "13:45", // another time (within schedule start_time and end_date)
+                    15, // another duration (>= service durations)
+                    $rId, // resource_id
+                    NULL // user_id defaults to self
+                );
+        }
+    }
+
+    /**
+     * Generate data from Booking response.
+     * @param  GonebusyLib\Models\EntitiesBookingsResponse $response
+     * @param  int $sId service ID (not in response)
+     * @param  int $rId resource ID (not in response)
+     * @return CreateBookingBody object with data from $response
+     */
+    private function bodyFromResponse($response, $sId, $rId) {
+        return new CreateBookingBody(
+            $response->booking->timeWindow->startDate, // date
+            $sId, // service_id
+            $response->booking->timeWindow->startTime, // time
+            $response->booking->timeWindow->totalMinutes, // duration
+            $rId, // resource_id
+            NULL // user_id defaults to self
+        );
+    }
+
+
+    /**
+     * Test POST /bookings/new
+     * GonebusyLib\Controllers\UsersController::createBooking()
+     */
+    public function testCreateBooking() {
+        $serviceResponse = $this->services->createService(Configuration::$authorization, $this->createBody('Service'));
+        $resourceResponse = $this->resources->createResource(Configuration::$authorization, $this->createBody('Resource'));
+        // Give open Schedule for seervice/resource combination:
+        $createScheduleBody = $this->scheduleBody($serviceResponse->service->id, $resourceResponse->resource->id);
+        $scheduleResponse = $this->schedules->createSchedule(Configuration::$authorization, $createScheduleBody);
+
+
+        // Create Booking:
+        $createBookingBody = $this->bookingBody('create', $serviceResponse->service->id, $resourceResponse->resource->id);
+        $response = $this->bookings->createBooking(
+            Configuration::$authorization,
+            $createBookingBody
+        );
+
+        // Was it created?
+        $this->assertInstanceOf('GonebusyLib\Models\CreateBookingResponse', $response);
+
+        // Does it have all the original data we sent?
+        $responseBody = $this->bodyFromResponse($response, $serviceResponse->service->id, $resourceResponse->resource->id);
+        $this->assertEquals($createBookingBody, $responseBody);
+
+        // Delete test booking after a few seconds:
+        sleep(3); // (until the booking is :awaiting_review)
+        $this->bookings->cancelBookingById(Configuration::$authorization, $response->booking->id);
```

Comment:
@alexagranov Here as well as in [line 235](https://github.com/gonebusy/gonebusy-php-client/pull/27/files#diff-98cf67d7a8840cc10dbf4b6d29009537R235) (in `testUpdateBookingById()`) I use `cancelBookingById()` to clean up after the test. Are bookings actually removed from the db though? Otherwise this is a waste of time since it involves a `sleep(3)` as discussed. Thanks